### PR TITLE
[AnomalyDetector] add cognitive service AnomalyDetector [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by moreOver0**
Our service contact already got signed off by review board #5126. But after that we had a rename, so remove the initial AnomalyFinder in PR #5294, then create this PR to commit all files for public preview and SDK generation.
Our service is still in process to go live, so do not merge this PR until we request to,

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - tag: release_1_0
+```
+
+### Tag: release_1_0 and go
+
+These settings apply only when `--tag=release_1_0 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'release_1_0' && $(go)
+output-folder: $(go-sdk-folder)/services/cognitiveservices/v1.0/$(namespace)
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md]: Please update this to `$(go-sdk-folder)/services/preview/cognitiveservices/v1.0/$(namespace)` as this is from a preview swagger.

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md (hunk 1 above)
Comment (moreOver0) [reply specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md]: What about other language?

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md]: Sorry I don't know, the other SDK language folks will have to comment.

**Comment by anuchandy** [comment]
@sarangan12 could you take a look at the ruby gen failure?

**Review by jhendrixMSFT — COMMENTED** [review]
Go SDK change looks good.

**Comment by moreOver0** [comment]
Hi @anuchandy, we have live service now. I think it's time to merge this PR, so that I can update the .Net sdk. ;)

**Comment by anuchandy** [comment]
@sarangan12 could help with ruby gen issue?
@shahabhijeet could you help with .NET?

**Comment by shahabhijeet** [comment]
startbuild

**Comment by shahabhijeet** [comment]
startbuild

**Comment by shahabhijeet** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/034b3543d5d04a6258111383e7979a0a79cad832

**Comment by shahabhijeet** [comment]
startbuild

**Comment by shahabhijeet** [comment]
startbuild

**Comment by shahabhijeet** [comment]
startbuild

**Comment by shahabhijeet** [comment]
startbuild

**Comment by shahabhijeet** [comment]
@anuchandy Jenkins is having issues. I have talked to the RP owner and this PR can be merged.
As there is an outstanding SDK PR that was opened 10 days back.
So the SDK is already getting generated. So once this PR is merged, I can merge the SDK PR

**Comment by shahabhijeet** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/034b3543d5d04a6258111383e7979a0a79cad832

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/034b3543d5d04a6258111383e7979a0a79cad832

**Review by anuchandy — APPROVED** [review]

**Comment by anuchandy** [comment]
sync with Ruby team and .Net team. 

For Ruby this is expected for brand new RPs. 
For .Net, the .net SDK PR based on this swagger has been already created so can ignore the CI failure.

**Comment by salameer** [comment]
Chatted with @anuchandy offline, Fire away guys :)

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**